modules/afsocket/afunix-source.c

```diff
@@ -85,9 +86,22 @@ static gboolean
 afunix_sd_init(LogPipe *s)
 {
   AFUnixSourceDriver *self = (AFUnixSourceDriver *) s;
+ 
+  if (!afsocket_sd_init_method(s))
+    return FALSE
+  
+  if (is_file_socket(self->filename))
+    {
+      afunix_sd_apply_perms_to_socket(self);
+    }
+  else
+    {
+      msg_warning("Source is not a unix socket",
+                evt_tag_str("filename", self->filename),
+                NULL);
```

Comment:
The previous version returned with FALSE when the file was not a socket...
Why did you change it?